packages/near-membrane-base/src/environment.ts

```diff
@@ -128,6 +159,18 @@ export class VirtualEnvironment {
         }
     }
 
+    link(path: string) {
+        const keys = path.split('.');
+        let bluePointer = this.blueGlobalThisPointer;
+        let redPointer = this.redGlobalThisPointer;
+        for (let i = 0, len = keys.length; i < len; i += 1) {
+            bluePointer = this.blueCallableGetPropertyValuePointer(bluePointer, keys[i]);
```

Comment:
another potential hazard here is the use of `__proto__`, instead of `getPrototypeOf`. This will require a new callable method to be exported, but it is doable.

Reply:
You mean using `getPrototypeOf` would require a new callable method to be exported? (not following this bit yet)

Reply:
yes, exactly that! because the one we have returns the actual proxy, but what we need here is a pointer directly, not the proxy. the reason why we need a pointer is because those are reflective objects, they don't need a proxy, they just map directly to the other side.